src/snowflake/cli/plugins/nativeapp/artifacts.py

```diff
@@ -283,3 +299,37 @@ def find_version_info_in_manifest_file(
             patch_name = version_info[patch_field]
 
     return version_name, patch_name
+
+
+def project_path_to_deploy_path(
+    project_paths: List[Path], files_mapping: DeployMapping
```

Comment:
nit: files_mapping is rather vague, consider a more descriptive name. You call that created_files in a different context too, and that's confusing to me because it's the same thing by a different name.